crates/update-notifier/src/ui.rs

```diff
@@ -0,0 +1,132 @@
+use colored::*;
+use terminal_size::{terminal_size, Width};
+
+mod utils;
+
+const PADDING: usize = 8;
+const TOP_LEFT: &str = "╭";
+const TOP_RIGHT: &str = "╮";
+const BOTTOM_LEFT: &str = "╰";
+const BOTTOM_RIGHT: &str = "╯";
+const HORIZONTAL: &str = "─";
+const VERTICAL: &str = "│";
+const SPACE: &str = " ";
```

Comment:
Is there a more rusty way of storing these?

Reply:
There's a dedicated single-character char type: https://doc.rust-lang.org/std/primitive.char.html